app/controllers/api/v1/users_controller.rb

```diff
@@ -9,7 +9,7 @@ class UsersController < ApiV1Controller
 
       def create
         # params = { auth: { provider: , uid: } }
-        return error_message unless params[:auth]
+        return render_message unless params[:auth]
 
         @user = User.from_omniauth(params.fetch(:auth))
         @token = TokenService.new(user: user).object
```

Comment:
use memoization to token or callbacks to create and show actions

Reply:
done